src/dotnet/devcontainer-template.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "dotnet",
-    "version": "1.0.2",
+    "version": "1.0.3",
```

Comment:
Should we bump this to 1.1.0 to follow more sem-ver conventions? Adding the new .NET 7 version as the default seems like a new feature worth a minor version bump.